### PR TITLE
Add alignof() info for "bessd -t"

### PR DESCRIPTION
**Pull request by sangjinhan**
also add `max_align_t` to the type list....

### CODE REVIEW

File: core/debug.cc [hunk 1]
```diff
@@ -429,35 +432,68 @@ void SetTrapHandler() {
   }
 }
 
+template <typename T>
```
Comment (barath) [inline core/debug.cc]: Function comment.

File: core/debug.cc [hunk 2]
```diff
@@ -429,35 +432,68 @@ void SetTrapHandler() {
...
-         sizeof(TrafficClass), sizeof(struct tc_stats));
-  printf("sizeof(Task)=%zu\n", sizeof(Task));
-
-  printf("sizeof(Module)=%zu\n", sizeof(Module));
-  printf("sizeof(Gate)=%zu\n", sizeof(bess::Gate));
-  printf("sizeof(IGate)=%zu\n", sizeof(bess::IGate));
-  printf("sizeof(OGate)=%zu\n", sizeof(bess::OGate));
-
-  printf("sizeof(worker_context)=%zu\n", sizeof(Worker));
+  std::cout << Format("gcc %d.%d.%d", __GNUC__, __GNUC_MINOR__,
```
Comment (barath) [inline core/debug.cc]: Function comment.

**Comment by sangjinhan** [comment]
Let me not add comments there; those functions are self explanatory, and never meant to be used anywhere else other than the only current user: `bessd -t`.

**Comment by changlan** [comment]
image

**Pull request merged**